tests/foreman/ui_airgun/test_location.py

```diff
@@ -27,3 +37,338 @@ def test_positive_create(session):
             'description': gen_string('alpha'),
         })
         assert session.location.search(loc_name)[0]['Name'] == loc_name
+
+
+def test_positive_delete(session):
+    loc_name = gen_string('alpha')
+    with session:
+        session.location.create({'name': loc_name})
+        session.location.delete(loc_name)
+        assert not session.location.search(loc_name)
+
+
+@run_only_on('sat')
+@tier2
+def test_positive_create_with_location_and_org(session):
```

Comment:
remove that test from both old ui and here as it doesn't make sense. We verify that functionality across application when select both org and loc